### DISCUSSION TITLE
Is there any way to obtain and de-drm Barnes and Noble books at this point?

### DISCUSSION THREAD

**Discussion by emtilt** [apprenticeharper/DeDRM_tools#2288] [General]
The windows app for Barnes and Noble has been discontinued as of the end of November. Is there any alternate procedure for accessing the purchased books and getting them into calibre without DRM?

**Comment by ElleKayEm** [comment]
People used to use rooted Android (or an emulator) with the Nook app or a rooted Nook.  The hardest part is getting your key.  But in my experience, a B&N account usually has one key which will work regardless of which app the books come from.  So if you previously got your key and still have it, the hard part may be done.  No guarantee this still works, but here is an old how-to on this method:  https://www.aricrenzo.com/2019-12-13-Liberate-Your-Nook-Ebooks/
